packages/commons/src/config/consumerServiceConfig.ts

```diff
@@ -1,16 +1,17 @@
 import { z } from "zod";
-import { AWSConfig } from "./awsConfig.js";
 import { KafkaConfig } from "./kafkaConfig.js";
 import { ReadModelDbConfig } from "./readmodelDbConfig.js";
 
-export const KafkaConsumerConfig = KafkaConfig.and(AWSConfig).and(
+export const KafkaConsumerConfig = KafkaConfig.and(
   z
     .object({
+      KAFKA_GROUP_ID: z.string(),
```

Comment:
KAFKA_GROUP_ID is an info specific to only consumers. 
So i moved it from the `KafkaConfig` and put it here in the `KafkaConsumerConfig` which is more fitting